plugins/ingest-geoip/src/main/java/org/elasticsearch/ingest/geoip/GeoIpProcessor.java

```diff
@@ -57,8 +57,8 @@
 public final class GeoIpProcessor extends AbstractProcessor {
 
     public static final String TYPE = "geoip";
-    private static final String CITY_DB_TYPE = "GeoLite2-City";
-    private static final String COUNTRY_DB_TYPE = "GeoLite2-Country";
+    private static final String CITY_DB_TYPE = "-City";
```

Comment:
Since this is now a suffix, can you change the constant name to reflect that, eg `CITY_DB_SUFFIX`? And same with country as well.